engine/src/main/scala/cromwell/engine/db/DataAccess.scala

```diff
@@ -370,9 +370,12 @@ trait DataAccess extends AutoCloseable {
     val key = metadataEvent.key
     val workflowUuid = key.workflowId.id.toString
     val timestamp = metadataEvent.timestamp
+    val value = metadataEvent.value.value
+    val valueType = if (metadataEvent.value.valueType == null) null else metadataEvent.value.valueType.typeName
```

Comment:
Didn't actually try it but I think this might work and would be DRYer:

`Option(metadataEvent.value.valueType) map { _.typeName } orNull`